core/flyway/src/test/java/io/thorntail/flyway/impl/FlywayConfigurationTest.java

```diff
@@ -0,0 +1,61 @@
+package io.thorntail.flyway.impl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+import java.util.NoSuchElementException;
+import java.util.Spliterators;
+
+import org.eclipse.microprofile.config.Config;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import io.thorntail.datasources.impl.DataSourceRegistry;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Unit tests are good, but this project seems to favor integration tests with the built in Thorntail runner.